package.json

```diff
@@ -104,7 +104,7 @@
     "inert": "5.1.0",
     "node-notifier": "5.2.1",
     "react": "16.2.0",
-    "react-async-bootstrapper": "1.1.2",
+    "react-async-bootstrapper": "2.1.0",
     "react-async-component": "1.0.2",
```

Comment:
You need to update to the latest `react-async-component` otherwise you will get deprecation warnings 👍